run_service_ssh.ps1

```diff
@@ -0,0 +1,35 @@
+# This script first runs the ggp ssh init --instance <id> command to get
+# the needed credentials to connect via ssh. Then connects to the instance 
+# with portforwarding and starts OrbitService.
+
+
+Invoke-Expression "ggp ssh init" | Tee-Object -Variable ggp_out
+
+$lines = $ggp_out -split "\n"
+
+ForEach ($line in $lines) {
+    if ($line.Contains("User:")) {
+        $ggp_user = $line.Replace("User:", "").Trim()
+    }
+    if ($line.Contains("Host:")) {
+        $ggp_host = $line.Replace("Host:", "").Trim()
+    }
+    if ($line.Contains("Port:")) {
+        $ggp_port = $line.Replace("Port:", "").Trim()
+    }
+    if ($line.Contains("Key Path:")) {
+        $ggp_key_path = $line.Replace("Key Path:", "").Trim()
+    }
+    if ($line.Contains("Known Hosts Path:")) {
+        $ggp_known_hosts_path = $line.Replace("Known Hosts Path:", "").Trim()
+    }
+}
+
+IF (!$ggp_user -Or !$ggp_host -Or !$ggp_port -Or !$ggp_key_path -Or !$ggp_known_hosts_path) {
+  throw "Unable to get all necessary information from ggp ssh init"
+}
+
+$ssh_command = "ssh -t -p`"$($ggp_port)`" -i`"$($ggp_key_path)`" -oStrictHostKeyChecking=yes -oUserKnownHostsFile=`"$($ggp_known_hosts_path)`" -L44766:localhost:44766 -L44755:localhost:44755 $($ggp_user)@$($ggp_host) -- sudo /mnt/developer/OrbitService"
```

Comment:
Can we assume that `ssh` is in the PATH on an installation that only contains GGP SDK and no Git For Windows?

Reply:
In my GGP_SDK folder there is an OpenSSH installation. (\tools\OpenSSH-Wind64\ssh.exe) 

I could use both executables (ssh and ggp) in relation to the GGP SDK. Do you think this is the best idea?

Reply:
Probably yes. I only know that we can't assume `ssh.exe` to be in the PATH.
Probably it's best to take `$Env:GGP_SDK_PATH` into account.

Reply:
done